config/default.yaml

```diff
@@ -8,6 +8,17 @@ webserver:
   https: false
   hostname: 'localhost'
   port: 9000
+  rates_limit:
```

Comment:
Create a dedicated section for `rates_limit` instead of adding it to the `webserver` section